.eslintrc.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
+  env: {
+    browser: false,
+    es6: true,
+    node: true,
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint'],
+  extends: ['prettier', 'prettier/@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/adjacent-overload-signatures': 'error',
```

Comment:
maybe we could go for using `plugin:@typescript-eslint/recommended` configuration, rather than configuring ourself each rules?
(and still overriding recommended config when it's not relally suited to what we want)

What do you think @vorpaljs-reforged/vorpal-developers ?

Reply:
i agree our config should only include our explicit overrrides